### DISCUSSION TITLE
HELP PLEASE why this is error?

### DISCUSSION THREAD

**Discussion by Votexz** [FunkinCrew/Funkin#2084] [Compiling help]
hi
im can't build kade engine 1.18 
but this is problem
even though I downloaded everything

_Error: Source path "C:/haxelib/extension-webm/git/ndll/Windows64/extension-webm.ndll" does not exist_

what doing?

PLEASE HELP

**Comment by alexandrucn** [comment]
do this:
haxelib remove extension-webm
haxelib install extension-webm
lime rebuild extension-webm windows
